cognite_toolkit/_cdf_tk/loaders/_resource_loaders/agent_loaders.py

```diff
@@ -22,7 +22,7 @@ class AgentLoader(ResourceLoader[str, AgentUpsert, Agent, AgentUpsertList, Agent
     list_write_cls = AgentUpsertList
     kind = "Agent"
     _doc_base_url = ""
-    _doc_url = "https://api-docs.cognite.com/20230101-alpha/tag/Agents/operation/main_api_v1_projects__projectName__ai_agents_post"
+    _doc_url = "https://api-docs.cognite.com/20230101-beta/tag/Agents/operation/main_ai_agents_post/"
```

Comment:
This is unrelated - but the tests starts failing immediately when the URL is no longer available - so it needs to be fixed to get the tests to pass.